### PR TITLE
GameBuddy v1.M

### PR DESCRIPTION
**Pull request by AlisGraveNil**
## Description

I created the GameBuddy Mirror Series version.
I moved the original GameBuddy files into a "rev1.0" folder and added the "rev1.M" folder for the Mirror version.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [X] Keyboard (addition or update)
- [X] Keymap/layout/userspace (addition or update)
- [X] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [X] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [X] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [X] I have added tests to cover my changes.
- [X] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/clawsome/gamebuddy/v1.M/gamebuddy_rev1M.c [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "GameBuddy2.M"
```
Comment (drashna) [inline keyboards/clawsome/gamebuddy/v1.M/gamebuddy_rev1M.c:17]:
```suggestion
#include "v1.m.h"
```

File: keyboards/clawsome/gamebuddy/v1.M/gamebuddy_rev1M.c [hunk 2]
```diff
@@ -0,0 +1,17 @@
+/* Copyright 2021 AAClawson (AlisGraveNil)
```
Comment (drashna) [inline keyboards/clawsome/gamebuddy/v1.M/gamebuddy_rev1M.c:1]: This file needs to match the folder name. 

So `v1.m.c`

File: keyboards/clawsome/gamebuddy/v1.M/gamebuddy_rev1M.h [hunk 3]
```diff
@@ -0,0 +1,33 @@
+/* Copyright 2021 AAClawson (AlisGraveNil)
```
Comment (drashna) [inline keyboards/clawsome/gamebuddy/v1.M/gamebuddy_rev1M.h:1]: This file needs to match the folder name. 

So `v1.m.h`

**Review by drashna — CHANGES_REQUESTED** [review]
The v1.0 folder would need to have the gamebuddy.c/h file names changed too.

And moving everything around like this means that this needs to target the develop branch, rather than the master branch.

**Comment by AlisGraveNil** [comment]
I have no idea what 'develop branch' means in this context; did I need to make a fork of a different master?

**Comment by AlisGraveNil** [comment]
@drashna - do I need to delete this PR to recreate it in the 'develop' branch?

**Comment by drashna** [comment]
https://docs.qmk.fm/#/breaking_changes_instructions

And no, you don't need to delete it. If you edit the title, it will let you choose a different branch to target in the QMK repo.  Select `develop`, and save.  After doing so, you may need to rebase your branch.  This guide should cover all of the rebasing stuff: 
https://docs.qmk.fm/#/newbs_git_best_practices

**Comment by AlisGraveNil** [comment]
@drashna - did I move it right?

**Comment by drashna** [comment]
Yup!

**Review by drashna — APPROVED** [review]

**Review by fauxpark — CHANGES_REQUESTED** [review]
`.`s in the keyboard file/folder names should be changed to `_`s.

**Comment by AlisGraveNil** [comment]
> `.`s in the keyboard file/folder names should be changed to `_`s.

I seem to have been able to rename all of the files except "1.M.c"
I tried completely renaming the file then going back, no dice.
Any suggestions?

File: keyboards/clawsome/gamebuddy/v1_M.M/v1_M.c [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "v1.M.h"
```
Comment (fauxpark) [inline keyboards/clawsome/gamebuddy/v1_M.M/v1_M.c:16]:
```suggestion
#include "v1_m.h"
```

File: keyboards/clawsome/gamebuddy/v1_0/v1_0.c [hunk 5]
```diff
@@ -13,4 +13,4 @@
  * You should have received a copy of the GNU General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-#include "gamebuddy.h"
\ No newline at end of file
+#include "v1.0.h"
```
Comment (fauxpark) [inline keyboards/clawsome/gamebuddy/v1_0/v1_0.c:16]:
```suggestion
#include "v1_0.h"
```

File: keyboards/clawsome/gamebuddy/v1.M.c [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "v1.M.h"
```
Comment (fauxpark) [inline keyboards/clawsome/gamebuddy/v1.M.c:17]:
```suggestion
#include "v1_m.h"
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by AlisGraveNil** [comment]
@fauxpark do I need to change "." to "_" in all files, or just the three you suggested changes to?

**Comment by fauxpark** [comment]
All files and folders must be lowercase, with no `.`s, and files that reference them (eg. `#include`s) need to match.

**Comment by AlisGraveNil** [comment]
@fauxpark - anything else I need to do on this?

File: keyboards/clawsome/gamebuddy/v1_M.M/config.h [hunk 7]
```diff
@@ -0,0 +1,45 @@
+/* Copyright 2021 AAClawson (AlisGraveNil)
```
Comment (fauxpark) [inline keyboards/clawsome/gamebuddy/v1_M.M/config.h:1]: This folder still needs to be renamed.

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/clawsome/gamebuddy/v1_M.M/config.h (hunk 7 above)
Comment (AlisGraveNil) [reply keyboards/clawsome/gamebuddy/v1_M.M/config.h]: I can't get it to rename - I got it to list "v1_m.M" but I can't get rid of the ".M" part of the folder name

Comment (fauxpark) [reply keyboards/clawsome/gamebuddy/v1_M.M/config.h]: Try using `git mv`.

**Comment by AlisGraveNil** [comment]
@fauxpark - I cannot get the folder to get rid of the ".M" suffix, no matter what I change the folder name to. Any suggestions?

**Comment by fauxpark** [comment]
I cloned your repo to try it myself - this seems to work fine:
```
git mv keyboards/clawsome/gamebuddy/v1_m.M keyboards/clawsome/gamebuddy/v1_m
```

**Comment by AlisGraveNil** [comment]
@fauxpark that worked! thank you!

File: keyboards/clawsome/gamebuddy/v1_M/config.h [hunk 8]
```diff
@@ -0,0 +1,45 @@
+/* Copyright 2021 AAClawson (AlisGraveNil)
```
Comment (fauxpark) [inline keyboards/clawsome/gamebuddy/v1_M/config.h:1]: Folder name still needs to be lowercase.

File: keyboards/clawsome/gamebuddy/v1_M/readme.md [hunk 9]
```diff
@@ -0,0 +1,13 @@
...
+
+This is a 5x7 macropad designed for left OR right hand usage in videogames, with special focus on FPS layouts!
+
+* Keyboard Maintainer: [AAClawson](https://github.com/AlisGraveNil)
+* Hardware Supported: GameBuddy, Pro Micro, Elite-C, nice!nano
+* Hardware Availability: www.clawboards.xyz
+
+Make example for this keyboard (after setting up your build environment):
+
+    make clawsome/gamebuddy1_M:default
```
Comment (fauxpark) [inline keyboards/clawsome/gamebuddy/v1_M/readme.md:11]:
```suggestion
    make clawsome/gamebuddy/v1_m:default
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/clawsome/gamebuddy/v1_M/config.h (hunk 8 above)
Comment (AlisGraveNil) [reply keyboards/clawsome/gamebuddy/v1_M/config.h]: folder name adjusted from "v1_M" to "v1_m"

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Comment by AlisGraveNil** [comment]
@drashna - now that this is merged into 'develop'... what does it do?

**Pull request merged**